### ISSUE TITLE
number-offset does not work with pdf output

### ISSUE THREAD

**Issue by thedabs91** [quarto-dev/quarto-cli#3126]
I tried to render pdf with number-offset. The document seemed to run, but produced the error message:

Aeson exception:
Error in $: parsing [] failed, expected Array, but encountered Number

When I changed output to html or removed number-offset, the code worked. A short .qmd script that produced the error is below:

```
---
title: "Title"
author: "Author"
format:
  pdf:
    number-sections: true
    number-offset: -1
---

## Introduction

## The Methods

### Loading Data

### Computing methods

## Results
```

FYI, I am using quarto on a 64-bit Windows 11 machine, and the LaTeX installation is tinytex.

**Comment by mcanouil** [comment]
Using the following notations allows to not have the error message, but the option is ignored in Quarto v1.2.253.

```
number-offset:
  - -1
```

```
number-offset: [-1]
```

**Comment by thedabs91** [comment]
Thank you. That helps. I will remember to enclose the number-offset in brackets when I use it, though it would be nice if I didn't need to. I checked that the brackets work with html as well, but I have not tested other formats.

**Comment by cscheid** [comment]
This is actually a bug on pandoc (and us by consequence)! We'll have a fix (a pandoc workaround) for 1.3, which should be out in a couple of weeks tops.

**Comment by cscheid** [comment]
upstream report: https://github.com/jgm/pandoc/issues/8410

**Comment by cscheid** [comment]
This is now fixed on our v1.3 branch. We don't currently have installers for that, but we should have them in a couple of weeks when v1.2 goes stable.
